bazel/linux/test.bzl

```diff
@@ -20,9 +97,13 @@ def qemu_test(name, kernel_image, run, qemu_binary = None,
         config: dict, all additional attributes to pass to the
             kernel_qemu_run rule, generally created with mconfig().
     """
+
+    runner_script = mcreate_rule(name, test_runner, "test-runner", [],
```

Comment:
with mcreate_rule, merge in kwargs. Otherwise `visibility = ` or `tag = ` will not propagate correctly to all rules instantiated by the macro.

Reply:
Fixed. I assume it just means I have to do `..., kwargs, mconfig(...`